packages/forms/src/utils/getCheckboxRadioStyles.ts

```diff
@@ -0,0 +1,20 @@
+const getCheckboxRadioStyles = () => ({
+  "data-h2-appearance": "base(none)",
+  "data-h2-background-color":
+    "base(white) base:focus-visible(focus) base:selectors[::before](primary) base:dark:selectors[::before](primary.light) base:all:selectors[:focus-visible::before](black)",
+  "data-h2-border": "base(thin solid black.light)",
+  "data-h2-color": "base(currentColor)",
+  "data-h2-content": "base:selectors[::before]('')",
+  "data-h2-display": "base(grid)",
+  "data-h2-flex-shrink": "base(0)",
+  "data-h2-place-content": "base(center)",
+  "data-h2-height": "base(x.9) base:selectors[::before](x.5)",
+  "data-h2-line-height": "base(x1)",
+  "data-h2-margin": "base(x.1 0 0 0)",
+  "data-h2-transform":
+    "base:selectors[::before](scale(0)) base:selectors[:checked::before](scale(1))",
+  "data-h2-transition": "base:selectors[::before](120ms transform ease-in-out)",
```

Comment:
Will this respect "prefer reduced motion"?

Reply:
Ooo, good call! I don't think it was. It should be now after [check reduced motion for animations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8976/commits/ace246226fd7f8ba15ac6d59806c9d1a497aa4ef)

Reply:
Great!